docs/source/options.rst

```diff
@@ -1863,6 +1863,14 @@ Type: int
 
 Default value: 10
 
+**CORE_GUESS**
```

Comment:
I think we need a better name for this. Core_guess does not make sense to me. Making it longer and clearer what it means.